extensions/database.py

```
@@ -6,85 +6,48 @@
-            self.user = None
+    async def init(self, socket: æ.WebSocket) -> namespace:
+        self.connection = await æ.aiomysql.connect(**æ.SUG.DBC)
         return self
     # CLASS -> QUERY
-    async def query(self, command: str, parameters: list) -> list[dict] | bool:
-        for index in range(len(parameters)):
-            if isinstance(parameters[index], dict | list):
-                parameters[index] = json.dumps(parameters[index])
+    async def query(self, command: str, parameters: list) -> æ.Any:
+        typeList = self.procedures[command]
+        for index in range(len(parameters)): assert isinstance(parameters[index], typeList[index])
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)